packages/protocol/contracts/mainnet/MainnetTierRouter.sol

```diff
@@ -7,8 +7,14 @@ import "../L1/tiers/ITierRouter.sol";
 /// @dev Labeled in AddressResolver as "tier_router"
 /// @custom:security-contact security@taiko.xyz
 contract MainnetTierRouter is ITierRouter {
+    uint256 public constant ONTAKE_FORK_HEIGHT = 367_200; // = 7200 * 52
```

Comment:
The result of the multiplication in the comment is 374_400.

Reply:
Fixed